### DISCUSSION TITLE
DraftOrder Add Line items Shopify API

### DISCUSSION THREAD

**Discussion by tariqbilal** [Kyon147/laravel-shopify#234] [Q&A]
I was wondering if I can add products on draft order via API. Is it possible any help would be great.

I tried adding a product but I am getting 

Client error: PUT https://{url}/admin/api/2023-10/draft_orders/DRAFT_ORDER_ID.jsonresulted in a400 Bad Request response:\n {"errors":{"draft_order":"Required parameter missing or invalid"}

**Comment by shirish71** [comment]
Can you provide the json that you are sending to Shopify?

**Comment by tariqbilal** [comment]
@shirish71 
```
$response = $shop->api()->rest('PUT', '/admin/api/2023-10/draft_orders/1140887159067.json', [
             "line_items" =>  [
                 [

                     "variant_id" => 8772320035099,
                     "quantity" => 1
                 ]
             ]
]);
````
